app/controllers/node_info_controller.rb

```diff
@@ -13,6 +13,9 @@ def document
   end
 
   def statistics
-    @statistics = NodeInfoPresenter.new("1.0")
+    respond_to do |format|
+      format.json { render nothing: true, status: 406 }
```

Comment:
Instead of `render nothing: true, status: 406` you can also do `head :not_acceptable`.